src/vcpkg.cpp

```diff
@@ -259,6 +260,11 @@ int main(const int argc, const char* const* const argv)
         }
     } // unlock g_metrics
 
+    if (metrics_enabled)
+    {
+        load_config(fs);
```

Comment:
This thing being called "load_config" and having lots of "spooky action at a distance" seems like a time bomb that is likely to regress.

How about changing the design such that metrics are off, then having metrics->enable_metrics() which has the load_config() guts inside?